### DISCUSSION TITLE
2023/05/01 Task

### DISCUSSION THREAD

**Discussion by hyunnnn98** [2P3S/scrum-dice#46] [General]
## 승현
**😄 UI 개발 시작!!**
* [x] #47 
* [x] #48 

## 재순
* https://github.com/2P3S/2p3s-backend-system/issues/1
	* [x] 채팅방 개설 기반 구현 -> 리뷰 @Samsam-lee @LeeJuYong0909 
	* [x] Mongo DB 통신 기반 구현 -> 리뷰 @Samsam-lee @LeeJuYong0909 
* https://github.com/2P3S/2p3s-backend-system/issues/9
	* [ ] Socket.io server 기반 착수

## 승형
* [ ] 사용자가 로그인했을 때, 어떤 방식으로 확인할 것인지 Sequence 디테일 수정
* [ ] ...

**Comment by JeongJaeSoon** [comment]
플래닝 포커 카드 종류
- 0, 1/2, 1, 2, 3, 5, 8, 13, 20, 40, 100
- 0, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89

**Reply by JeongJaeSoon** [reply]
불투명 블러 배경
https://blogpack.tistory.com/1014
